anki/template/template.py

```diff
@@ -4,6 +4,9 @@
 from anki.hooks import runFilter
 from anki.utils import stripHTML, stripHTMLMedia
 
+# The (?si) flags make the regex match case-insensitively and make . match any
```

Comment:
I don't think it makes sense to add comments that just repeat the Python docs, especially as these flags are used throughout the code.  Otherwise looks good - thanks :-)

Reply:
Done.